packages/react-vapor/src/components/calendar/Calendar.tsx

```diff
@@ -257,21 +269,42 @@ export class Calendar extends React.Component<ICalendarProps, any> {
             'calendar',
             {
                 'mod-width-50': !this.props.simple,
+                'countdown-calendar': this.props.countdown,
             },
             this.props.wrapperClassNames
         );
 
+        if (this.props.countdown) {
+            this.countdownHeader = (
+                <div id="countdown-header">
+                    <h2 className="bold mb1">
+                        {moment().date(moment().month(this.props.months[selectedMonth]).daysInMonth()).fromNow(true)}{' '}
+                        left
+                    </h2>
+                    <div className="smaller">in {this.props.months[selectedMonth]}</div>
```

Comment:
I think smaller is too small for the month :see_no_evil: 
![image](https://user-images.githubusercontent.com/63734941/112301439-511ccd80-8c70-11eb-9138-f75ffc0a70d5.png)

Reply:
I'm inclined to agree but I went by the mockup https://coveord.atlassian.net/browse/ADUI-6728